app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -207,7 +214,7 @@ public void onClick(View view) {
                     } else {
                         Globals.CurrentMatchNumber = Integer.parseInt(preMatchBinding.editMatch.getText().toString());
                         Globals.NumberMatchFilesKept = Globals.sp.getInt(Constants.SP_NUM_MATCHES, 5);
-                        Globals.CurrentTeamToScout = Integer.parseInt(spinner_Team.getSelectedItem().toString());
+                        Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
 
```

Comment:
same thing.  you can't assign the POSITION to the teamToScout.  It worked before, why did you change it?